src/Rector/Deprecation/Base/EntityViewBase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation\Base;
+
+use DrupalRector\Utility\AddCommentTrait;
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+
+/**
+ * Replaced deprecated entity_load() calls.
+ *
+ * See https://www.drupal.org/node/2266845 for change record.
+ *
+ * What is covered:
+ * - Static replacement
+ * - The reset parameter is handled with a ternary operator
+ *
+ * Improvement opportunities
+ * - Dependency injection
+ */
+abstract class EntityViewBase extends AbstractRector
+{
+
+  use AddCommentTrait;
```

Comment:
Did you want to add a comment to the replacement about checking for the `reset` option?

If not, we could remove this trait.